karate-tests/src/test/java/helpers/utils.feature

```diff
@@ -62,6 +62,39 @@ Scenario:
   }
   """
 
+  * def randomTempAddress =
+  """
+  function() {
```

Comment:
Is there any way we can reuse some of the existing and similar code already available under the randomAddress function above? It would be good to reuse as much as possible.

Reply:
I don't think we can reuse much `randomAddress` generates address without period end date based on provided earliest start date

`randomTempAddress` generates a temporary address with period end date based on current date

Reply:
I've had a closer look, it would be cumbersome to try and inherit from one util function to another within utils.feature . I would suggest, time permitting, to consider moving some of this functionality to Utils.java where we have more flexibility (as a tech debt task/sub-task).